### PR TITLE
New package: apcupsd-3.14.14

### PR DESCRIPTION
**Pull request by CaseOf**
Hello,
I made a template for apcupsd and thought it was a good idea to share it.

### CODE REVIEW

File: srcpkgs/apcupsd/template [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	etc/apcupsd/apcupsd.conf
+	etc/apcupsd/hosts.conf
+	etc/apcupsd/multimon.conf
+	etc/apcupsd/apcupsd.css
+	etc/apcupsd/changeme
+	etc/apcupsd/commfailure
+	etc/apcupsd/commok
+	etc/apcupsd/offbattery
+	etc/apcupsd/onbattery
+	etc/apcupsd/apccontrol"
```
Comment (abenson) [inline srcpkgs/apcupsd/template]: These need to absolute paths with the leading /, iirc. You could do `conf_files="/etc/apcupsd/*"` as well

File: srcpkgs/apcupsd/template [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ --sysconfdir=/etc/apcupsd
+ --with-pwrfail-dir=/etc/apcupsd
+ --with-upstype=usb --with-upscable=usb --enable-usb --with-dev=
+ --enable-modbus-usb
+ --enable-net --enable-pcnet
+ --enable-snmp
+ --disable-gapcmon
+ --with-nis-port=3551"
+hostmakedepends="autoconf automake"
+makedepends="libusb-compat libusb-compat-devel net-snmp net-snmp-devel man-pages man-pages-devel"
```
Comment (abenson) [inline srcpkgs/apcupsd/template]: You just need the `-devel` packages. You probably don't need the man-pages package.

File: srcpkgs/apcupsd/template [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	for bin_file in apcaccess apcupsd apctest smtp; do
+		vbin src/${bin_file}
+	done
+	vmkdir etc/${pkgname}
+	vinstall platforms/apccontrol 744 etc/${pkgname}
+	for conf_file in apcupsd.conf hosts.conf multimon.conf; do
+		vinstall platforms/etc/${conf_file} 644 etc/${pkgname}
+	done
+	for conf_file in changeme commfailure commok offbattery onbattery; do
+		vinstall platforms/etc/${conf_file} 744 etc/${pkgname}
```
Comment (abenson) [inline srcpkgs/apcupsd/template]: What's up with these permissions?

File: srcpkgs/apcupsd/files/apcupsd/run [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+install -d -m 0775 -o root -g uucp /run/apcupsd
```
Comment (abenson) [inline srcpkgs/apcupsd/files/apcupsd/run]: I'm unsure of this, too

File: srcpkgs/apcupsd/template (hunk 3 above)
Comment (CaseOf) [reply srcpkgs/apcupsd/template]: These scripts are only meant to be executed as root so…

File: srcpkgs/apcupsd/files/apcupsd/run (hunk 4 above)
Comment (CaseOf) [reply srcpkgs/apcupsd/files/apcupsd/run]: how would you check if a directory exists?

Comment (abenson) [reply srcpkgs/apcupsd/files/apcupsd/run]: also group `uucp`

Comment (CaseOf) [reply srcpkgs/apcupsd/files/apcupsd/run]: changed

File: srcpkgs/apcupsd/files/apcupsd/run [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -d /run/apcupsd ] || mkdir /run/apcupsd
```
Comment (abenson) [inline srcpkgs/apcupsd/files/apcupsd/run]: probably want -p

Comment (CaseOf) [reply srcpkgs/apcupsd/files/apcupsd/run]: if the item is a file and not a dir, mkdir is failing even with -p.
i also assume /run always exist, else system is probably broken…

File: srcpkgs/apcupsd/template [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum="db7748559b6b4c3784f9856561ef6ac6199ef7bd019b3edcd7e0a647bf8f9867"
+
+pre_configure() {
+	export SHUTDOWN=/usr/bin/shutdown
+	export WALL=/usr/bin/wall
+	export APCUPSD_MAIL=/usr/bin/mail
+}
+
+do_install() {
```
Comment (Piraty) [inline srcpkgs/apcupsd/template:32]: any reason not to make use of the default `make install`  and do cleanup in post_install if required ?

**Pull request closed without merging**